docs/uml-class-diagrams/display02/production/DisplayComm2WayInterfaceImplementation/DisplayComm2WayInterfaceImplementation.puml

```diff
@@ -2,12 +2,13 @@
 
 package edu.oakland.production.display02 {
     class DisplayComm2WayInterfaceImplementation {
+        DisplayCommManager manager
         __
+        {abstract} + DisplayComm2WayInterfaceImplementation(DisplayCommManager manager)
         .. Use Case 1 ..
-        {abstract} + LocationDataPoint passRfidRequest(int rfidNum)
-
-        .. Use Case 2 ..
-
+        {abstract} + TrackData passRfidRequest(int rfidNum)
+        {abstract} + void determineUserRequestFormat(int rfidNum)
+        {abstract} + void receiveRequestFromDisplayCommInterface(int rfidNum)
```

Comment:
Remember: you're implementing this here, so it shouldn't be abstract. Also, are these private? If so, they should have the minus sign instead of the `+` sign.